cmd/awsinventory/awsinventory.go

```diff
@@ -61,10 +59,11 @@ func main() {
 
 	// Write stored rows to csv inventory
 	var count int
-	awsData.MapRows(func(row inventory.Row) error {
-		count++
-		return csv.WriteRow(row)
-	})
+	awsData.Load(regions, services, nil)
+	// awsData.Load(regions, services, func(row inventory.Row) error {
```

Comment:
Please remove if it is no longer needed.

Reply:
Sorry about that! I've switched that back now and added proper handling for that situation. That should only be set to `nil` during testing when we don't care about the rows, for example, when testing that an error is returned.

Reply:
The reason I didn't notice the hanging is that the rows channel can only hold 100 rows before blocking and the account I tested against had less than 100 resources in it :roll_eyes: